### ISSUE TITLE
why loss suddenly become the initial value

### ISSUE THREAD

**Issue by maozhiqiang** [NVIDIA/waveglow#38]
123873: -5.480178833
123874: -5.352289200
123875: -5.298225403
123876: -5.460500240
123877: -5.547176361
123878: -5.597653866
123879: -5.451864243
123880: -5.445433617
123881: -5.494832516
123882: -5.272130013
123883: -5.312953949
123884: -5.550226212
123885: -5.573468208
123886: -5.405485153
123887: -4.718036175
123888: -5.176541805
123889: -4.795463085
123890: -4.772330284
123891: -4.845523357
123892: 40490.179687500
123893: -3.482588530
123894: -2.216503620
123895: -1.672664404
123896: -1.454238176
123897: -1.419931889
123898: -1.281533122
123899: -1.110476017
123900: -1.149755478
123901: -1.464488387
123902: -1.522264838
123903: -1.716608882
123904: -1.713886499

**Comment by rafaelvalle** [comment]
Please share the config.json file.

**Comment by maozhiqiang** [comment]
{
    "train_config": {
        "output_directory": "checkpoints",
        "epochs": 100000,
        "learning_rate": 1e-4,
        "sigma": 1.0,
        "iters_per_checkpoint": 2000,
        "batch_size": 3,
        "seed": 1234,
        "checkpoint_path": ""
    },
    "data_config": {
        "training_files":"train_files.txt",
        "segment_length": 16000,
        "sampling_rate": 16000,
        "filter_length": 1024,
        "hop_length": 256,
        "win_length": 1024,
        "mel_fmin": 0.0,
        "mel_fmax": 8000.0
    },
    "dist_config": {
        "dist_backend": "nccl",
        "dist_url": "tcp://localhost:54321"
    },

    "waveglow_config": {
        "n_mel_channels": 80,
        "n_flows": 12,
        "n_group": 8,
        "n_early_every": 4,
        "n_early_size": 2,
        "WN_config": {
            "n_layers": 8,
            "n_channels": 512,
            "kernel_size": 3
        }
    }
}
thanks!

**Comment by rafaelvalle** [comment]
I see your sampling rate is 16kHz and assume you are not using LJS.
The loss could have such a spike if your dataset has an audio sample that is all "silence".

**Comment by maozhiqiang** [comment]
@rafaelvalle thanks! I will check that

**Comment by rafaelvalle** [comment]
Closing to discuss on related issue.
